src/getTimezoneAPI.js

```diff
@@ -0,0 +1,6 @@
+const fetch = require("node-fetch");
+
+exports.getTimezone = function getTimezone(clientIP){
+    const clientInfo = fetch(`https://ipapi.co/${clientIP}/json`)
+    return clientInfo.utc_offset || "+0800" //Default : Hong Kong
+}
```

Comment:
UTC offset and timezone is two different thing. Please return the timezone instead and set it with the date time library (https://github.com/moment/luxon).